activities/codebase-stewardship/supporting-governance/governance-template.md

```diff
@@ -9,89 +9,89 @@ The text below can be used as a template for creating a governance file in a rep
+* ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles
 
 ## Technical steering team
 
-The [codebase] technical steering team members are active contributors who are on a day-to-day basis responsible for:
-
-* Merging pull requests
-* Handling code of conduct violations
-* Overseeing the resolution and disclosure of security issues
+[codebase]'s technical steering team members are active contributors whose day-to-day activities include:
 
-If technical steering team members cannot reach consensus informally, the question at hand should be forwarded to the technical steering team meeting.
+* merging pull requests
+* handling code of conduct violations
+* overseeing the resolution and disclosure of security issues
 
-Any active member of the community can request to become a technical steering team member by asking the technical steering team. The technical steering team will vote on it (simple majority).
+Together, the technical steering team is responsible for:
 
-The current team members are:
+* providing technical direction for the codebase
+* maintaining a technical roadmap, an architecture and coding principles
+* resolving issues in development or conflicts between contributors
+* managing and planning releases
+* controlling rights to [codebase] assets such as source repositories
 
-*
-
-The technical steering team has the joint responsibility to:
-
-* Provide technical direction for the codebase
-* Maintain a technical roadmap, an architecture and coding principles
-* Resolve issues in development or conflicts between contributors
-* Managing and planning releases
-* Controlling rights to [codebase] assets such as source repositories
+Ideally, no one company or organization will employ a simple majority of the technical steering team.
 
-The technical steering team meets regularly. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly being done through the process described in CONTRIBUTING.md [add link].
+Any active member of the community can request to become a technical steering team member by asking the technical steering team. Members are appointed by the current technical steering team members.
 
-If more than one technical steering team member disagrees with a decision of the technical steering team they may appeal to the product steering team who will make the ultimate ruling.
+### Working process
 
-Ideally, no one company or organization will employ a simple majority of the technical steering team.
+The technical steering team meets regularly [explain frequency]. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly done through the process described in CONTRIBUTING.md [add link].
```

Comment:
The `CONTRIBUTING.md` file should not be called that here, it should be called the 'contributing guide' as in other codebases it might have a different file name or so.

See next note on frequency.